concent_api/core/models.py

```diff
@@ -28,6 +34,29 @@
 from .constants             import MESSAGE_TASK_ID_MAX_LENGTH
 
 
+class SubtaskWithTimingColumnsManager(Manager):
+    """Creates maximum_download time, subtask_verification_time and download_deadline columns
+    maximum_download_time = DOWNLOAD_LEADIN_TIME + ceil((result_package_size / MINIMUM_UPLOAD_RATE << 10))
+    subtask_verification_time = (4 * CONCENT_MESSAGING_TIME) + (3 * maximum_download_time) + (0.5 * (computation_deadline - task_to_compute_timestamp))
+    download_deadline = computation_deadline + maximum_download_time
+    """
+    def get_queryset(self):
+        bytes_per_sec = settings.MINIMUM_UPLOAD_RATE << 10
```

Comment:
I suggest separate with blank lines calculation of all three columns if possible. And add a comment on the start of each calculation.